src/components/mediainfo/mediainfo.js

```diff
@@ -193,14 +202,14 @@ define(['datetime', 'globalize', 'appRouter', 'itemHelper', 'indicators', 'mater
 
                     try {
 
-                        var endYear = datetime.parseISO8601Date(item.EndDate).getFullYear();
+                        const endYear = datetime.parseISO8601Date(item.EndDate).getFullYear();
 
                         if (endYear !== item.ProductionYear) {
-                            text += '-' + datetime.parseISO8601Date(item.EndDate).getFullYear();
+                            text += `-${datetime.parseISO8601Date(item.EndDate).getFullYear()}`;
                         }
 
                     } catch (e) {
-                        console.error('error parsing date: ' + item.EndDate);
+                        console.error(`error parsing date: ${item.EndDate}`);
```

Comment:
Hmmm, I don't know. @MrTimscampi @ferferga maybe we want to start using literals for error messages?

Reply:
Seems good. We can also us `console.error('error parsing date:', item.EndDate);` and browsers should handle it.

Reply:
Do you think we should enforce one style over the other for most instances though? I think picking one would improve readability when parsing console lines in the code.

Reply:
I think we should. Also make use of more advanced console utilities like groups, asserts, counts, traces, etc.

We can improve our readability and the usefulness of our logs a lot, I feel.